### DISCUSSION TITLE
Workaround for Slinger player / Android

### DISCUSSION THREAD

**Discussion by bradut** [GerryDazoo/SlingerPlayer#18] [Show and tell]
Slinger player for Android works great on my phone.

Unfortunately it doesn't work on my tablet and on MINIX TV box, where it only displays the  flame logo, and then stops. 
I've seen that some people here had the same issue.

Looking for a fix,I've experimented the following workaround:
- Installed on the tablet an Android virtual machine, and imported the SP app from host (tablet).
(Initially I imported the APK but the VM didn't allow me to install.)

Overall, the experience is not great, as the VM is slow, probably due to the low memory (3GB RAM) , and allows Slinger player work only in portrait mode.
But, the most important thing is that it works !

The VM I tried is "Virtual Android", and I may experiment with different ones to see if I will find a faster one.

Hope this helps

**Comment by as400jockey** [comment]
Why run under a VM?  I have run the apk on my android tablet (android 11) and it works great.

A VM sounds like overhead that you don’t need…

I have run the Windows version on my desktop and my laptop and it works great.

**Reply by bradut** [reply]
I've explained in a different thread that on my tablet (Android 8.0), the player doesn't work.
I could install it, and after the first start, asked for permissions then got stuck into displaying the flame logo.
Same issue happened when I installed on a MINIX Android box for TV.

It does work great on my Android 11 phone.

So, while waiting for my plane to take off, I gave it a try with the VM.
Will do more experiments in 24 hours when I reach my destination.

**Reply by bradut** [reply]
Update, 
I tried a few VMs and it seems to me that 3 GB of RAM are not enough to run a VM decently, so I decided not to follow this path.

In the meantime I found out that my tablet can run the Slinger player, but, unlike on the phone,  it didn't display the URL selector unless I tapped on the "flame" screen. 
Something is not quite right with my tablet as it runs the app very slowly, and, on landscape mode, video freeze within a few minutes. 
By comparison, the app works great on a much older Samsung phone.

**Reply by sp222222** [reply]
well, I cannot get it to run on android 5.1.1 OS :) sdk is too old it says when sideloaded from a winPC.

It does work on an old phone running android 9 however :)

**Reply by bradut** [reply]
UPDATE:
Solved by flipping the tablet into portrait mode to see the menu, same as we use it on the phone. (Probably it wold have helped if there were a message in the main screen indicating this.).

...didn't cross my mind at the right moment
